doc/manual/installation/upgrading.xml

```diff
@@ -7,15 +7,21 @@
   <title>Upgrading Nix</title>
 
   <para>
-    Multi-user Nix users on macOS can upgrade Nix by running
-    <command>sudo -i sh -c 'nix-channel --update &amp;&amp; nix-env
-    -iA nixpkgs.nix'; sudo launchctl stop org.nixos.nix-daemon; sudo
-    launchctl start org.nixos.nix-daemon</command>.
+    Multi-user Nix users on macOS can upgrade Nix by running this:
+    <screen>$ sudo -i sh -c 'nix-channel --update &amp;&amp; nix-env
+      -iA nixpkgs.nix'; sudo launchctl stop org.nixos.nix-daemon; sudo
+      launchctl start org.nixos.nix-daemon</screen>
```

Comment:
We should keep `<command>` but I don't mind dropping the period, and switching to the `running this:` part.